spec/system/admin/order_cycles/complex_updating_specific_time_spec.rb

```diff
@@ -81,7 +81,7 @@
     open_all_exchange_product_tabs
 
     expect(page).to have_selector
-      "#order_cycle_incoming_exchange_1_variants_#{initial_variants.last.id}"
+    "#order_cycle_incoming_exchange_1_variants_#{initial_variants.last.id}"
```

Comment:
This is interesting. Does this mean that the string is not seen as method argument? Oh, I can't test it because the whole spec is disabled. This is dead and broken code. Apparently this spec was commented out as flaky three years ago:

* 042b4faf1e05fa73ba72886d23b44fae1fcf9ab2

@filipefurtad0 Can you asses if we should fix the spec or delete it?

Reply:
Nice, thanks for digging that out. I would keep it and create an issue to keep track of it - I did so [here](https://github.com/openfoodfoundation/openfoodnetwork/issues/11266).

I've tried to fix it, but it's not a quick win, it's difficult to get a consistent behavior in the OC pages (the `add Distributor` button...)... Anyway, out of scope here.